runtime/compiler/runtime/RelocationRuntime.hpp

```diff
@@ -311,7 +311,7 @@ class TR_RelocationRuntime {
          TR_ASSERT(g >= 0 && g < TR_NumGlobalValueItems, "invalid index for global item");
          _globalValueList[g] = v;
          }
-      static char *       nameOfGlobal(uint32_t g)
+      static const char *       nameOfGlobal(uint32_t g)
```

Comment:
I think the method name `nameOfGlobal` used to begin in the same column as `setGlobalValue` and `getGlobalValue`.  May I ask you to remove the extra spaces?